### ISSUE TITLE
If a queen dies, new larvas cannot evolve into a drone.

### ISSUE THREAD

**Issue by DominikPanic** [tgstation/TerraGov-Marine-Corps#243]

**Comment by Rohesie** [comment]
This commit introduced this bug: <https://github.com/tgstation/TerraGov-Marine-Corps/commit/3cb388c558be50102f5c6527b3429e7ae6024782>
Mainly here:
```
else if(!hive.living_xeno_queen && potential_queens == 1 && isXenoLarva(src) && new_caste_type == /mob/living/carbon/Xenomorph/Drone)
	to_chat(src, "<span class='xenonotice'>The hive currently has no sister able to become Queen! The survival of the hive requires you to be a Drone!</span>")
	return
```
It should be `new_caste_type != /mob/living/carbon/Xenomorph/Drone`

**Comment by DominikPanic** [comment]
Fixed in #393
